reward/reward_config_cache_test.go

```diff
@@ -92,11 +92,10 @@ func (governance *testGovernance) StakingUpdateInterval() uint64 {
 	return governance.stakingInterval
 }
 
-func (governance *testGovernance) setTestGovernance(epoch uint64, mintingAmount string, ratio string, unitprice uint64, useGiniCoeff bool, deferredTxFee bool) {
+func (governance *testGovernance) setTestGovernance(epoch uint64, mintingAmount string, ratio string, useGiniCoeff bool, deferredTxFee bool) {
```

Comment:
@kjeom Please undo the removal of unitPrice as we've discussed in the [comment](https://github.com/klaytn/klaytn/pull/1409/files#r888543015). Don't forget to undo both `reward_config_cache_test.go` and `reward_distributor_test.go`.

Reply:
Thanks for remind!